src/components/database.rs

```diff
@@ -14,13 +14,41 @@ use crate::entities::{
 
 pub type DBConnection = Pool<Postgres>;
 
+#[cfg_attr(test, faux::create)]
 #[derive(Clone)]
```

Comment:
remove the derive

Reply:
It's needed because of app usage, it's being cloned for the health components